### PR TITLE
Markdown Card: Fix not rendering on initial load

### LINKED ISSUE #5861: Markdown Card not rendering: Beta 0.110.0b0
##  Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

Markdown Card not rendering any text until edited.

image

30997695f8e534eea5b20cc2feaa02d4

## Expected behavior

To show the Text in the card

## Steps to reproduce

1. Create Markdown Card
2. Add Text to it
3. Finish the edit
4. Refresh browser

## Environment

- Home Assistant release with the issue: 0.110.0b0
- Last working Home Assistant release (if known): 0.109.6
- Browser and browser version: Chrome and Firefox (latest)
- Operating system: Linux and Window

## Problem-relevant configuration

```yaml
type: markdown
content: >-
  The **Markdown** card allows you to write any text. You can style it **bold**,
  *italicized*, ~strikethrough~ etc. You can do images, links, and more.


  For more information see the [Markdown
  Cheatsheet](https://commonmark.org/help).
```

## Javascript errors shown in your browser console/inspector

Weirdly I have no console error

## Additional information

I would think this may have to do with the new ways views are handled. With @bramkragten recent changes.

Will take a look more into it later. Couple hours if I get to it tonight

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5861
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by zsarnett** [comment]
I made a few more changes than needed but I looked at it and couldn't not do it. It May not even be correct but looked like it needed to be changed. Feel free to push to the branch if needed in the morning. As I wont be available and not sure if you want to do a Beta push before then

**Comment by zsarnett** [comment]
Tested on multiple devices in Browserstack.

**Review by iantrich — APPROVED** [review]

File: src/panels/lovelace/cards/hui-markdown-card.ts [hunk 1]
```diff
@@ -58,19 +58,17 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
...
-      }
+      this._connect();
     });
   }
 
-  public disconnectedCallback() {
-    this._disconnect();
+  public connectedCallback() {
+    super.connectedCallback();
+    this._connect();
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-markdown-card.ts:67]: Do we always have `hass` and config when connected?

**Review by bramkragten — APPROVED** [review]

File: src/panels/lovelace/cards/hui-markdown-card.ts (hunk 1 above)
Comment (zsarnett) [reply src/panels/lovelace/cards/hui-markdown-card.ts:67]: We check in the connect function

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-markdown-card.ts:67]: I know, but if we don't have `hass` then, we will never connect.

Comment (zsarnett) [reply src/panels/lovelace/cards/hui-markdown-card.ts:67]: So you are saying add it to first updated as well?

**Pull request merged**